README.md

```diff
@@ -11,19 +11,19 @@ npm install -g @aragon/cli
 ## Usage
 
 ```
-➜ aragon-example-module aragon-dev-cli --help
+➜ aragon-example-application aragon-dev-cli --help
```

Comment:
We should rename to `aragon-example-app`, 'application' makes the name too long. I would encourage use of 'app' instead of 'application' for most instances.

Reply:
The repo is called `-application`. We can change it here, but not sure if we can rename the repo because we might have a link to it somewhere.